bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=1,
+                        class_naming_camel_case: bool=True,
+                        spaces_before_colon: int=-1,
+                        spaces_after_colon: int=-1,
+                        enforce_newline_at_EOF: bool=True,
+                        use_spaces: bool=True,
```

Comment:
Reading the param description, I feel it's better to use `no_tabs` (also used by linter as naming). There are already enough params that deal with spacing here like space before after operator, space before after arrows, braces etc. Enough spaces to confuse already. Also the name `use_spaces` appears little vague to me. Let's simplify it and use `no_tabs`? :stuck_out_tongue: